cypress/plugins/buildProject.js

```diff
@@ -1,13 +1,14 @@
 const { join } = require("path");
-const { spawnSync } = require("child_process");
+const execa = require("execa");
 
 // Build the given NextJS project
 const buildProject = ({ project }, config) => {
   process.stdout.write(`Building project: ${project}...`);
 
   // Build project
-  spawnSync("npm", ["run", "build"], {
+  execa.sync("npm", ["run", "build"], {
```

Comment:
`npm` is a shell script and calling `spawn()` on it does not work as intended on Windows without the `shell` option.
`execa` handles this case automatically though, providing with Windows support.